packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1126,19 +1126,41 @@ class ScriptPopupMenuOption {
 }
 
 final defaultScriptPopupMenuOptions = [
-  copyScriptNameOption,
+  copyPackageOption,
+  copyFilePathOption,
   goToLineOption,
   openFileOption,
 ];
 
-final copyScriptNameOption = ScriptPopupMenuOption(
-  label: 'Copy filename',
+final copyPackageOption = ScriptPopupMenuOption(
+  label: 'Copy package path',
   icon: Icons.content_copy,
   onSelected: (_, controller) => Clipboard.setData(
     ClipboardData(text: controller.scriptLocation.value?.scriptRef.uri),
   ),
 );
 
+final copyFilePathOption = ScriptPopupMenuOption(
+  label: 'Copy file path',
+  icon: Icons.content_copy,
+  onSelected: (_, controller) async {
+    String? filePath;
+    final packagePath = controller.scriptLocation.value!.scriptRef.uri;
+    if (packagePath != null) {
+      final results = await serviceManager.service!.lookupResolvedPackageUris(
+        serviceManager.isolateManager.selectedIsolate.value!.id!,
+        [packagePath],
+      );
+      if (results.uris != null) {
+        filePath = results.uris![0];
+      }
+    }
+    return Clipboard.setData(
+      ClipboardData(text: filePath),
```

Comment:
What if filePath is null here? We do not want to copy null to clopboart, right?
I suggest to replace ifs with asserts if we know for sure packagePath and results.uris are not nulls.

Reply:
I had thought about this but noticed the original copy didn't take it into account either. 
I purposely let this fall through to null so that we are no worse off than the original behaviour.

From my understanding asserts don't work in a fully built app, would those asserts just be there for our testing purposes?

Reply:
> From my understanding asserts don't work in a fully built app, would those asserts just be there for our testing purposes?

Correct. Asserts only run in debug mode.